examples/use_rust/nrf52840_ble_split/src/peripheral.rs

```diff
@@ -129,6 +136,7 @@ async fn main(spawner: Spawner) {
     let storage_config = StorageConfig {
         start_addr: 0x60000, // 384K
         num_sectors: 32,     // 128K
+        clear_storage: true, // Ensure clean ble peer addr
```

Comment:
I prefer not to add this because it will make the peripheral clear saved central on every reboot. For new users it might be confusing.

This option should only be added when needed, for example, upgrading from old version to newer. Having a comment on it could be great.